### ISSUE TITLE
[CoE Starter Kit - QUESTION] 

### ISSUE THREAD

**Issue by Chitusnair** [microsoft/coe-starter-kit#8258]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Will the flows and apps for which the owners are Service Accounts be marked as Orphaned apps/flows by CoE?

### What solution are you experiencing the issue with?

Core

### What method are you using to get inventory and telemetry?

None

**Comment by ClaudioRWS** [comment]
and oh man,

I'm going to use the example I have here in my tenant, no matter how long the app or flow has the service account, the flow searches for inactivity.

What you can do is mark the environment to not run the automation


![image](https://github.com/microsoft/coe-starter-kit/assets/94450591/3d39f08d-3ea3-4bda-bd5d-5ec0fd79786f)

**Comment by Chitusnair** [comment]
Thanks, However, my question is around Orphaned apps and Flow rather than inactivity. In the Set Flow permission app, I have noticed that flows which are having SVC account as owners are marked as Orphaned.

**Comment by Jenefer-Monroe** [comment]
No, service accounts behave as regular accounts, and are present from the O365 connect just as a regular user. 
For service principles (app registrations) these are also correctly shown as non-orphan as we do know how to look for those as users too.

**Comment by Chitusnair** [comment]
Thanks for the reply. So if any Inactivity alerts / Compliance emails etc will be sent to the Service account mail box? Is it possible to send it to Co-owner in such cases? We want users to follow the best practice of Service accounts as Owners of the resources, but not sure how to manage the communications to an actual user. (using service principal is not an option for us now in place of SVC accounts)

**Comment by Jenefer-Monroe** [comment]
Hello, today there is no such concept in the kit, but we do have a feature area in place to reconsider "ownership" in the kit. 
I can add this to that for consideration when we get there. Target is early autumn.

**Comment by AmarSaiRam** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: https://github.com/https://github.com/microsoft/coe-starter-kit/issues/10329

Thank you for your contributions
